src/Address/Address.php

```diff
@@ -31,13 +31,13 @@ class Address implements AddressInterface, GeocodableInterface
 
     /**
      * @Assert\Length(max=15, groups={"Default", "Update"})
-     * @Assert\NotBlank(groups={"Update"})
+     * @Assert\Expression(expression="this.getCountry() !== 'FR' or value", message="common.city_name.not_blank", groups={"Update"})
```

Comment:
should we use the constant `self::FRANCE` for the `FR` value ?

Reply:
fixed